mixer/adapter/kubernetesenv/kubernetesenv.go

```diff
@@ -51,7 +51,7 @@ const (
 	podServiceLabel                    = "app"
 	istioPodServiceLabel               = "istio"
 	lookupIngressSourceAndOriginValues = false
-	istioIngressSvc                    = "ingress.istio-system.svc.cluster.local"
+	istioIngressSvc                    = "istio-ingress.istio-system.svc.cluster.local"
```

Comment:
what happens when istio is installed in some other namespace ? or someone changes the name of the svc ? or the cluster domain ? [I realize those questions predate the pr]

Reply:
Yes. This doesn't fix pre-existing issues with derivation, just makes what is there work consistently.